{{ cookiecutter.project_slug }}/docker/development/docker-compose.yml

```diff
@@ -21,10 +21,21 @@ services:
             dockerfile: docker/development/Dockerfile
         volumes:
             - "../..:/app"
+            - "{{ cookiecutter.project_slug }}-node_modules:/app/node_modules"
         ports:
             - "8000:8000"
         networks:
             - {{ cookiecutter.project_slug }}
         depends_on:
             - {{ cookiecutter.project_slug }}-db
         command: "/app/tools/run-development.sh"
+        environment:
+            {{ cookiecutter.project_slug_upper }}_DB_NAME: {{ cookiecutter.project_slug }}
+            {{ cookiecutter.project_slug_upper }}_DB_USER: {{ cookiecutter.project_slug }}
+            {{ cookiecutter.project_slug_upper }}_DB_PASSWORD: secret
+            {{ cookiecutter.project_slug_upper }}_DB_HOST: {{ cookiecutter.project_slug }}-db
+            {{ cookiecutter.project_slug_upper }}_DB_PORT: 5432
```

Comment:
Hmmm, could we think about adding this cool `django-dj` or what's it's name which parses the database URL into the Django settings? It's much more convenient to configure containers on AWS (or wherever) and specify only one secret instead of five ;)

Reply:
https://github.com/jacobian/dj-database-url - that one? I'm all in.

Reply:
Hm, maybe I will add whitenoise too, seems like we are using the package often recently as well? What do you think?

Reply:
Exactly this one, let's there be disco:

![](https://uploads.freegogpcgames.com/image/Disco-Elysium.jpg)

Reply:
> Hm, maybe I will add whitenoise too, seems like we are using the package often recently as well? What do you think?

Yup, good idea.